controllers/pipeline/pipeline_controller_test.go

```diff
@@ -262,6 +262,7 @@ var _ = Describe("PipelineController", func() {
 					},
 				},
 			}
+
```

Comment:
Is it expected?

Reply:
No. Thanks, it is added unintentionally.